doc/api/net.md

```diff
@@ -1642,6 +1622,47 @@ Use `nc` to connect to a Unix domain socket server:
 $ nc -U /tmp/echo.sock
 ```
 
+## `net.getDefaultAutoSelectFamily()`
```

Comment:
Should the function names be alphabetically ordered? Is there are any requirements?

Reply:
Nope, AFAIK there's no strict requirement.
But my slight OCD kicked in and I think it's a good habit.